pages/common/tsc.md

```diff
@@ -0,0 +1,28 @@
+# tsc
+
+> TypeScript compiler.
+> More information: <https://www.typescriptlang.org/docs/handbook/compiler-options.html>.
+
+- Compile a TypeScript file `hello.ts` into a JavaScript file `hello.js`:
+
+`tsc {{foo.ts}}`
+
+- Compile a TypeScript file into JavaScript using a specific target syntax (default is `ES3`):
+
+`tsc --target {{ES5|ES2015|ES2016|ES2017|ES2018|ESNEXT}} {{foo.ts}}`
+
+- Compile a TypeScript file into a JavaScript file with a custom name:
+
+`tsc --outFile {{foo.js}} {{bar.ts}}`
```

Comment:
Does --outFile {{bar.js}} {{foo.ts}} sound better ? So that the foo -> bar direction is maintained.

Reply:
Sure, makes sense.

Reply:
Actually, since this is precisely an example to show usage of custom names, let's go with something more explicit: "input.ts" and "output.js".

Reply:
Sure